wagtailio/core/models/snippets.py

```diff
@@ -2,14 +2,8 @@
 
 from modelcluster.fields import ParentalKey
```

Comment:
the utils snippets are not related to content page. Can you please undo this change?
Since we will ultimately replace the existing menu relates snippets, the move is just unnecessary

Reply:
Thanks, will do :+1: